src/components/ButtonWithDropdownMenu.js

```diff
@@ -88,6 +97,9 @@ const ButtonWithDropdownMenu = (props) => {
                         isDisabled={props.isDisabled}
                         style={[styles.pl0]}
                         onPress={() => {
+                            if (popoverAnchorPosition === null) {
+                                measurePopoverPosition();
+                            }
                             setIsMenuVisible(true);
                         }}
```

Comment:
Rewrite this as:
1. Measure popover position
2. Set popover position (inside measure callback)
3. Set menu visible (also inside measure callback)

Reply:
@s77rt 3 other instances of popover are already fixed in another commit, https://github.com/Expensify/App/commit/bbd951af5ba346f60fd06ba1a00e439c781a047c

Reply:
I think you are asking me to make this callback function

```
    const measure = () => {
        if (popoverAnchorPosition === null) {
            measurePopoverPosition();
        }
        setIsMenuVisible(true);
    }
```

and call it in onPress of caretButton, is it correct?

Reply:
I meant something like this:
```js
        caretButton.current.measureInWindow((x, y, w, h) => {
            setPopoverAnchorPosition({
                horizontal: x + w,
                vertical: y + h,
            });
            setIsMenuVisible(true);
        });
```

Reply:
okay thank you I just did it like you mensioned here and pushed my changes, please check it and let me know if we can make anything better